src/content/blog/2023/05/03/react-canaries.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "React Canaries: Enabling Incremental Feature Rollout Outside Meta"
+---
+
+May 3, 2023 by [Dan Abramov](https://twitter.com/dan_abramov), [Sophie Alpert](https://twitter.com/sophiebits), [Rick Hanlon](https://twitter.com/rickhanlonii), [Sebastian Markbåge](https://twitter.com/sebmarkbage), and [Andrew Clark](https://twitter.com/acdlite)
+
+---
+
+<Intro>
+
+Traditionally, new React features used to only be available at Meta first, and land in the open source releases later. We'd like to offer the React community an option to adopt individual new features as soon as their design is close to final--similar to how Meta uses React internally. We are introducing a new officially supported [Canary release channel](/community/versioning-policy#canary-channel). It lets curated setups like frameworks decouple adoption of individual React features from the React release schedule.
```

Comment:
> used to only be available at Meta first, and land in the open source releases later

Could add something like "...once they had been thoroughly tested in a real-world application" so people don't think the motivation was to privilege Meta or be withholding.